### PR TITLE
Adds try_vcs_checkout to wheel_generation workflow.

### PR DESCRIPTION
**Pull request by agalbachicar**
# 🦟 Bug fix

Follow up to https://github.com/maliput/maliput_infrastructure/pull/316

## Summary
The wheel generation workflow was not switching branches based on the naming scheme. Doing it here so it gets fixed.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by agalbachicar** [comment]
I think there is a problem with the Bazel build. It could be related to flags.
Note the integration tests: 
```
...
//test/integration:integration_test_town01                               PASSED in 92.7s
...
//test/integration:integration_test_figure8                             TIMEOUT in 900.0s 
```

And then, when checking the colcon tests:
```
...
INTEGRATION_extensive_queries_test_Town01.xodr ..........................   Passed    6.69 sec
...
INTEGRATION_extensive_queries_test_Figure8.xodr .........................   Passed   70.03 sec
```

**Comment by francocipollone** [comment]
Updating branch to bring last changes and re-trigger CI

File: .github/workflows/wheel_generation.yml [hunk 1]
```diff
@@ -37,10 +38,14 @@ jobs:
       with:
         path: ${{ env.COLCON_WS }}/src/${{ env.PACKAGE_NAME }}
     # clone public dependencies
-    - name: git clone maliput
+    - name: vcs import
+      shell: bash
+      working-directory: ${{ env.COLCON_WS }}
+      run: PATH=$PATH:${{ env.VCS_PATH }} vcs import src < src/${PACKAGE_NAME}/.github/dependencies.repos
```
Comment (francocipollone) [inline .github/workflows/wheel_generation.yml:44]: Note that `dependencies.repos` is also bringing ament_cmake_doxygen which is not necessary in this workflow. When doing `colcon build` I am not sure if there will be an issue for this package as we are not running rosdep install before due to obvious reasons and we are installing the dependencies manually.

**Review by francocipollone — APPROVED** [review]
LGTM!

**Pull request merged**